visualization/motion_planning_tasks/src/task_display.cpp

```diff
@@ -189,7 +189,12 @@ void TaskDisplay::changedRobotDescription() {
 }
 
 inline std::string getUniqueId(const std::string& process_id, const std::string& task_id) {
-	return process_id + "/" + task_id;
+	std::string id{ process_id };
```

Comment:
Has this change any semantic benefit? It's just about having trailing slash or not in case of an empty task_id, isn't it?
I would prefer the simpler code over this marginal/questionable improvement.